extra_geom/detectors.py

```diff
@@ -2026,6 +2026,13 @@ def _tile_slice(cls, tileno):
         fs_slice = slice(tile_fs_offset, tile_fs_offset + cls.frag_fs_pixels)
         return ss_slice, fs_slice
 
+    @classmethod
+    def _module_coords_to_tile(cls, slow_scan, fast_scan):
+        row, tile_fs = np.divmod(fast_scan, cls.frag_fs_pixels)
+        column, tile_ss = np.divmod(slow_scan, cls.frag_ss_pixels)
+        tileno = (column * 4 + row).astype(np.int16)
```

Comment:
```suggestion
        column, tile_fs = np.divmod(fast_scan, cls.frag_fs_pixels)
        row, tile_ss = np.divmod(slow_scan, cls.frag_ss_pixels)
        tileno = (row * 4 + column).astype(np.int16)
```

The logic is fine, but the variable names are swapped.

Reply:
yes, makes sense. thanks!